deliver/lib/deliver/languages.rb

```diff
@@ -1,7 +1,126 @@
+require_relative 'upload_metadata'
+
 module Deliver
   module Languages
     # 2020-08-24 - Available locales are not available as an endpoint in App Store Connect
     # Update with Spaceship::Tunes.client.available_languages.sort (as long as endpoint is avilable)
     ALL_LANGUAGES = ["ar-SA", "ca", "cs", "da", "de-DE", "el", "en-AU", "en-CA", "en-GB", "en-US", "es-ES", "es-MX", "fi", "fr-CA", "fr-FR", "he", "hi", "hr", "hu", "id", "it", "ja", "ko", "ms", "nl-NL", "no", "pl", "pt-BR", "pt-PT", "ro", "ru", "sk", "sv", "th", "tr", "uk", "vi", "zh-Hans", "zh-Hant"]
   end
 end
+
+module Deliver
+  class PrepareLanguages
+    attr_reader :enabled_languages
+    attr_reader :app_store_version_localizations
+    attr_reader :app_info_localizations
+
+    def prepare!(options)
```

Comment:
I appreciate if you can leave one-line comments for public methods or documentation! 🙏